pytorch_vision_fcn_resnet101.md

```diff
@@ -18,7 +18,9 @@ order: 10
 
 ```python
 import torch
-model = torch.hub.load('pytorch/vision:v0.10.0', 'fcn_resnet101', pretrained=True)
+model = torch.hub.load('pytorch/vision:v0.10.0', 'fcn_resnet50', pretrained=True)
```

Comment:
was the change from 101 to 50 intended?

I would suggest to leave 101 because when running the example from the colab notebook, 101 ends up with a more accurate segmentation

Reply:
I think it's because of its popularity. ResNet50 ended up being used much more in applications and papers because it balances accuracy for speed. Given it's the most popular across the resnet v1 variants, it's not a bad idea to highlight this instead of the deeper version.

Reply:
My idea was to download model with lower bandwidth. `resnet50` takes slightly less time in inference and has smaller model footprint. Thus speeding up CI slightly.

Reply:
If it's just about CI, even 10 seconds for a single won't matter much. But the popularity of the variant makes sense. Thanks for the replies